Patterns/English/English-DateTime.yaml

```diff
@@ -251,7 +251,7 @@ DateExtractor4: !nestedRegex
   def: \b{MonthNumRegex}\s*[/\\\-]\s*{DayRegex}[\.]?\s*[/\\\-]\s*{DateYearRegex}
   references: [ MonthNumRegex, DayRegex, DateYearRegex ]
 DateExtractor5: !nestedRegex
-  def: \b{DayRegex}\s*[/\\\-\.]\s*({MonthNumRegex}|{MonthRegex})\s*[/\\\-\.]\s*{DateYearRegex}
+  def: \b{DayRegex}\s*[/\\\-\.]\s*({MonthNumRegex}|{MonthRegex})\s*[/\\\-\.]\s*{DateYearRegex}(?!\s*[/\\\-\.]\s*\d+)
```

Comment:
My understanding is that this is making the regex not match if some suffix exists, correct? If so, this seems a reasonable solution. 
However, I think it would be important to add a comment in the YAML file explaining the reason for this component in the regex. This will greatly serve future maintainability and extension to new languages.
Also, this fix seems to target a general problem, so it should be made across languages too. Not only English (at least in languages were such entity mention form is plausible).

Reply:
yes, correct. 
Ok, we will add the comment and check the behaviour in the other languages.

Reply:
Do you have an estimate on when you can update the PR? It's getting little stale and now has a conflict with /master.

Reply:
@tellarin, just pushed the update